### DISCUSSION TITLE
polling_interval_seconds & mqtt update

### DISCUSSION THREAD

**Discussion by razorback91** [jagheterfredrik/wallbox-mqtt-bridge#55] [Q&A] [answered]
Hello,

I have defined 1 in the polling_interval_seconds option in the config file but I have mqtt update for powerboost only every about 10s. Is it possible to have more frequent info from the powerboot or am I doing something wrong ?

avril 13 19:59:55 wallbox_221859/power_boost_power_l1/state 560.5
avril 13 20:00:06 wallbox_221859/power_boost_power_l1/state 611.4
avril 13 20:00:17 wallbox_221859/power_boost_power_l1/state 575.6
avril 13 20:00:28 wallbox_221859/power_boost_power_l1/state 580.9
avril 13 20:00:39 wallbox_221859/power_boost_power_l1/state 609.4
avril 13 20:00:50 wallbox_221859/power_boost_power_l1/state 578.9

Thank you

Xavier

**Comment by jagheterfredrik** [comment, marked as answer]
See https://github.com/jagheterfredrik/wallbox-mqtt-bridge/discussions/48#discussioncomment-8820153

**Reply by razorback91** [reply]
I did a mqtt bridge and indeed I receive now lot of info
